deployments/dsa_plugin/base/intel-dsa-plugin.yaml

```diff
@@ -33,6 +33,13 @@ spec:
             type: "container_device_plugin_t"
           readOnlyRootFilesystem: true
           allowPrivilegeEscalation: false
+        resources:
+          requests:
+            memory: "25Mi"
+            cpu: 40m
+          limits:
+            memory: "50Mi"
+            cpu: 50m
```

Comment:
I'd increase the cpu limit to maybe 60m?

Reply:
basically DLB uses less than DSA. May I ask why you think it is better to set DLB's 80m and DSA's 60m?

Reply:
No real reason. Pod consuming 60m or 80m of CPU is so small that it doesn't really matter which it is. I would personally "play it safe" just so Pods don't get evicted/deleted due to too low limits. I'm not sure if CPU limits cause that but memory surely will cause the Pod to get deleted.

Reply:
@tkatila OK, thanks.!
I made even safer (but based on the collected data).!